docs/guidance/build/change_history.md

```diff
@@ -12,43 +12,13 @@ For an introduction to the concept of a change history, see the [releases and re
 For guidance on how to set the `id` field on each release, see the [easy releases](easy_releases) worked example.
 ```
 
-## Planning
-
-The London Borough of Barnet plans to publish a tender for cycle lane improvements later in the year. To prepare the market they publish a *notice of planned procurement*. This is also known as *prior information notice*, or *future opportunity notice*.
-
-The publisher creates an OCDS release to represent this notice. The release uses the 'planning' tag and includes data about the planned procurement. The `tender` section holds the estimated values and publication date. The tender status is 'planned'.
-
-The publisher also creates an OCDS record for the new contracting process. The releases list includes the new and only release so far. The compiled and versioned releases are also created for the record. Since there is only one release, the compiled version is expected look very much like this release. Compare the planning release and the record using the box below.
-
-```{jsoninclude} ../../examples/planning.json
-:jsonpointer: /releases
-:expand: planning, tag, tender
-:title: release
-```
-
-```{jsoninclude} ../../examples/records/planning.json
-:jsonpointer: /records
-:expand: compiledRelease, planning, tender
-:title: record
-```
-
 ## Tender
 
-The London Borough of Barnet is ready to invite bids for the contract. They issue the tender via an *notice of intended procurement*. This is also known as *contract notice* or *opportunity notice*.
-
-The publisher creates a new release to represent this notice. The release uses the 'tender' tag and provides the details in the `tender` section.
-
-There are no changes to the planning release published before. But the new release has the updated information about the tender. This includes the actual publication date, and the following new information:
-
-* The deadline for bid submission
-
-* The items to buy
-
-* Links to copies of tender documents
+The London Borough of Barnet publishes a tender for cycle lane improvements. They issue the tender via an *notice of intended procurement*. This is also known as *contract notice* or *opportunity notice*.
 
-* The estimated tender value
+The publisher creates a release to represent this notice. The release uses the 'tender' tag and provides the details in the `tender` section.
 
-The publisher adds the new release to the record, in the releases list. Also they update the compiled and versioned releases with the new information. Note that the bid submission date has changed in the compiled release. Also, note that the versioned release has a list of changes for each field that has been updated.
+The publisher adds the release to the record, in the releases list. Also they update the compiled and versioned releases with the new information.
```

Comment:
I think we want the (edited) paragraph from the old Planning section.

```suggestion
The publisher also creates a record for the new contracting process. The releases list includes the new and only release so far. The compiled and versioned releases are also created for the record. Since there is only one release, the compiled release is nearly identical to this release. Compare the release and the record using the box below.
```